### ISSUE TITLE
Review the structures introduced by GC lookups feature and server mode

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#2042]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/2038
- Created at 2013-07-30 09:43:41 by [dpal](https://pagure.io/user/dpal)
- Closed at 2020-03-24 14:11:37 as wontfix
- Assigned to nobody
---

Document SDAP interface.

### Comments

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2013-07-30 12:54:43](https://pagure.io/SSSD/sssd/issue/2038#221546)**

Fields changed

milestone: Interim Bucket => SSSD 1.12 beta

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2013-08-01 15:52:15](https://pagure.io/SSSD/sssd/issue/2038#221547)**

Fields changed

rhbz:  => 0

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2014-05-30 16:05:34](https://pagure.io/SSSD/sssd/issue/2038#221548)**

Should be done with the LDAP refactoring planned for 1.13

milestone: SSSD 1.12 beta => SSSD 1.13 beta

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2014-09-16 10:51:24](https://pagure.io/SSSD/sssd/issue/2038#221549)**

The LDAP provider is in a bad need for refactoring.

priority: major => blocker

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2014-09-16 10:54:18](https://pagure.io/SSSD/sssd/issue/2038#221550)**

Fields changed

blockedby:  => 2434

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2014-09-26 18:49:58](https://pagure.io/SSSD/sssd/issue/2038#221551)**

Fields changed

mark:  => 0

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2015-02-10 13:35:52](https://pagure.io/SSSD/sssd/issue/2038#221552)**

It would be nice to document the sdap interface, but it's probably out of scope for 1.13

milestone: SSSD 1.13 beta => SSSD 1.13 backlog
priority: blocker => major

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2015-02-12 20:12:57](https://pagure.io/SSSD/sssd/issue/2038#221553)**

Mass-moving tickets not planned for the 1.13 release to 1.14

milestone: SSSD 1.13 backlog => SSSD 1.14 beta

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2016-06-20 11:50:03](https://pagure.io/SSSD/sssd/issue/2038#221554)**

The sdap_ops.c code and the LDAP provider must be refactored, but it's too late for 1.14.

milestone: SSSD 1.14 beta => SSSD 1.15 beta
sensitive:  => 0

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2016-06-27 14:55:44](https://pagure.io/SSSD/sssd/issue/2038#221555)**

We should also use internal error codes to avoid propagating generic error codes up to the failover _recv functions. If we use internal error codes, then the topmost errors could be more descriptive.

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2017-02-24 14:24:00](https://pagure.io/SSSD/sssd/issue/2038#221556)**

**Metadata Update from @dpal**:
- Issue marked as depending on: #2434
- Issue set to the milestone: SSSD Future releases (no date set yet)

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2020-03-13 13:49:56](https://pagure.io/SSSD/sssd/issue/2038#633158)**

**Metadata Update from @thalman**:
- Custom field blocking adjusted to 2434
- Custom field design_review reset (from 0)
- Custom field mark reset (from 0)
- Custom field patch reset (from 0)
- Custom field review reset (from 0)
- Custom field sensitive reset (from 0)
- Custom field testsupdated reset (from 0)
- Issue close_status updated to: None
- Issue tagged with: Canditate to close

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2020-03-24 14:11:35](https://pagure.io/SSSD/sssd/issue/2038#636648)**

Thank you for taking time to submit this request for SSSD. Unfortunately this issue was not given priority and the team lacks the capacity to work on it at this time.

Given that we are unable to fulfill this request I am closing the issue as wontfix.

If the issue still persist on recent SSSD you can request re-consideration of this decision by reopening this issue. Please provide additional technical details about its importance to you.

Thank you for understanding.

---

**Comment from [dpal](https://pagure.io/user/dpal) at [2020-03-24 14:11:37](https://pagure.io/SSSD/sssd/issue/2038#636649)**

**Metadata Update from @pbrezina**:
- Issue close_status updated to: wontfix
- Issue status updated to: Closed (was: Open)
